src/ddsketch.rs

```diff
@@ -222,25 +222,108 @@ mod tests {
         }
 
         // We expect the following mappings from quantile to value:
-        // [0,0.33]: 1.0, (0.34,0.66]: 2.0, (0.67,0.99]: 3.0, (0.99, 1.0]: 4.0
+        // [0,0.25]: 1.0, (0.25,0.5]: 2.0, (0.5,0.75]: 3.0, (0.75, 1.0]: 4.0
         let test_cases = vec![
             (0.0, 1.0),
             (0.25, 1.0),
-            (0.33, 1.0),
-            (0.34, 2.0),
+            (0.26, 2.0),
             (0.5, 2.0),
-            (0.66, 2.0),
-            (0.67, 3.0),
+            (0.51, 3.0),
             (0.75, 3.0),
-            (0.99, 3.0),
+            (0.76, 4.0),
             (1.0, 4.0),
         ];
 
         for (q, val) in test_cases {
             assert_relative_eq!(dd.quantile(q).unwrap().unwrap(), val, max_relative = alpha);
         }
     }
+    #[test]
+    fn test_negative_quartiles() {
+        let alpha = 0.01;
+        let c = Config::new(alpha, 2048, 10e-9);
+        let mut dd = DDSketch::new(c);
+
+        // Initialize sketch with {1.0, 2.0, 3.0, 4.0}
```

Comment:
Update. Or remove since the comment's not really necessary. Can be updated in a few places below too